app/models/post_stat.rb

```diff
@@ -15,8 +17,13 @@ class PostStat < ActiveRecord::Base
 #  composer_open_duration_msecs :integer
 #  created_at                   :datetime         not null
 #  updated_at                   :datetime         not null
+#  composer_version             :string
```

Comment:
Should this be an `integer` now? Looks like we use both at different times for `*_version` fields.

Reply:
Ah yeah could be better as integer now, then it matches `baked_version` on post :+1: